host/4/bullseye/amd64/python/python37/python37.Dockerfile

```diff
@@ -34,9 +34,23 @@ RUN apt-get update && \
     rm -f /$EXTENSION_BUNDLE_FILENAME_V3 &&\
     find /FuncExtensionBundles/ -type f -exec chmod 644 {} \;
 
-FROM python:3.7-slim-bullseye
+FROM mcr.microsoft.com/dotnet/runtime-deps:6.0
```

Comment:
We can optimize these Python images once they are updated.  Optimization will allow us to share layers more effeciently in the platform.

Reply:
cc @paulbatum - python images are moving to the common base layer as other images !